index.js

```diff
@@ -27,46 +21,11 @@ module.exports = function derbyBundler(app, options) {
   }
   const { derby } = app;
   const { App, util } = derby;
+  debugger;
```

Comment:
Remove lingering `debugger`

Normally this should be caught by eslint, but looks like eslint hasn't been set up for this repo yet